bindings/c/test/mako/mako.c

```diff
@@ -743,7 +745,11 @@ int run_one_transaction(FDBTransaction* transaction,
 	for (i = 0; i < MAX_OP; i++) {
 
 		if ((args->txnspec.ops[i][OP_COUNT] > 0) && (i != OP_TRANSACTION) && (i != OP_COMMIT)) {
-			for (count = 0; count < args->txnspec.ops[i][OP_COUNT]; count++) {
+			int ops_count = args->txnspec.ops[i][OP_COUNT];
+			if (random() % 100 < 20) {
+				ops_count *= (random() % 9 + 2);
```

Comment:
Why not start from 1?

Reply:
Because I thought If we start from 1, it doesn't increase the commit size indeed. 
Does it make sense? 
let me make it to start from 1 for now.